libfuse/dir.go

```diff
@@ -347,6 +353,12 @@ func (d *Dir) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.Lo
 	d.folder.fs.log.CDebugf(ctx, "Dir Lookup %s", req.Name)
 	defer func() { d.folder.reportErr(ctx, libkbfs.ReadMode, err) }()
 
+	err = libkbfs.EnterCriticalWithTimeout(
```

Comment:
There are no matching `ExitCritical` calls for any of these.  Shouldn't there be, to clean up state?